code/game/machinery/doors/airlock.dm

```diff
@@ -555,6 +555,8 @@
 				light_state = AIRLOCK_LIGHT_BOLTS
 			else if(emergency)
 				light_state = AIRLOCK_LIGHT_EMERGENCY
+			if(!greyscale_config)
+				. += get_airlock_overlay("[airlock_material]_closed", icon , src, em_block = FALSE)
```

Comment:
s a bit unfortunate that we can't just give these states to greyscales but it costs dev time/init time so like whatever